src/test/java/teammates/test/cases/action/InstructorEditStudentFeedbackSaveActionTest.java

```diff
@@ -659,6 +663,19 @@ protected String getPageResultDestination(
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        StudentAttributes student = typicalBundle.students.get("student1InCourse1");
+
+        String feedbackSessionName = "First feedback session";
+        String courseId = student.course;
+        String moderatedStudentEmail = student.email;
+
+        String[] submissionParams = new String[]{
+                Const.ParamsNames.COURSE_ID, courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON, moderatedStudentEmail
+        };
+
+        verifyOnlyInstructorsOfTheSameCourseCanAccess(submissionParams);
+        verifyUnaccessibleWithoutModifyCoursePrivilege(submissionParams);
```

Comment:
`verifyUnaccessibleWithoutModifySessionCommentInSectionsPrivilege` is more appropriate here.